src/components/common/BuildStatus.test.js

```diff
@@ -20,6 +20,7 @@ describe('BuildStatus', () => {
       Running: '.fa-refresh',
       Succeeded: '.text-success'
     };
+    // eslint-disable-next-line no-unused-vars
```

Comment:
@Boomatang Why do we disable the `no-unused-vars` check?

Reply:
There was an issue with eslint saying there was usnused-vars in the for loops. We have found a fix. babel-eslint@10.0.3 needs to be add. I am doing the update now.